bower.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "cedar",
-  "version": "0.0.0",
+  "version": "0.4.0",
```

Comment:
Remove this line. version has been deprecated from bower, it just uses the releases available on github.